pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -5,11 +5,23 @@ CREATE SCHEMA IF NOT EXISTS SCHEMA_CATALOG; -- catalog tables + internal functio
 CREATE SCHEMA IF NOT EXISTS SCHEMA_PROM; -- data tables + public functions
 CREATE SCHEMA IF NOT EXISTS SCHEMA_SERIES;
 CREATE SCHEMA IF NOT EXISTS SCHEMA_METRIC;
+CREATE SCHEMA IF NOT EXISTS SCHEMA_DATA;
 
 
 CREATE EXTENSION IF NOT EXISTS timescaledb WITH SCHEMA public;
 
 CREATE DOMAIN SCHEMA_PROM.label_ids AS int[] NOT NULL;
+
+DO $$
+DECLARE
+   new_path text;
+BEGIN
+   new_path := current_setting('search_path') || format(',%L,%L', 'SCHEMA_PROM', 'SCHEMA_METRIC');
+   execute format('ALTER DATABASE %I SET search_path = %s', current_database(), new_path);
```

Comment:
I'm not sure how this is going to interact with cloud and whether this will be allowed through Aiven...but we'll deal with that when the time comes.